template/default.tmpl

```diff
@@ -31,6 +31,11 @@
 {{ define "pagerduty.default.instances" }}{{ template "__text_alert_list" . }}{{ end }}
 
 
+{{ define "victorops.default.entity_id" }}{{ .GroupLabels.SortedPairs.Values | join " " }} {{ if gt (len .CommonLabels) (len .GroupLabels) }} / {{ with .CommonLabels.Remove .GroupLabels.Names }}{{ .Values | join " " }}{{ end }}{{ end }}{{ end }}
+{{ define "victorops.default.message" }}{{ template "__subject" . }} | {{ template "__alertmanagerURL" . }}{{ end }}
+{{ define "victorops.default.from" }}{{ template "__alertmanager" . }}{{ end }}
```

Comment:
This field returneds the name of the tool from which the incident came from. 
VictorOps displays this information then. 
![b](https://cloud.githubusercontent.com/assets/10563183/16657796/0c1f904a-4464-11e6-9406-819ce3a8ae0c.png)

To avoid misunderstanding I can rename this field into the json key VictorOps expects (monitoring_tool), described here: [VictorOps API](http://victorops.force.com/knowledgebase/articles/Getting_Started/Incident-Fields-Glossary/?q=Create+Incident&l=en_US&fs=Search&pn=1).